sdk/Pulumi.Tests/Resources/CustomTimeoutsTests.cs

```diff
@@ -13,13 +13,13 @@ public void TestDeserialize()
             var rpc = new Pulumirpc.RegisterResourceRequest.Types.CustomTimeouts
             {
                 Create = "1m",
-                Update = "1h3m12.99ms100us",
+                Update = "1h3m12.99ms1us100ns",
```

Comment:
Good catch, I'd really like this fix pulled into its own PR with a changelog entry. We can merge it very quickly.